components/Form/Form.jsx

```diff
@@ -0,0 +1,313 @@
+import Image from 'next/image'
+import Modal from 'react-modal'
+import { useState } from 'react'
+import { RadioButton } from '@/components'
+import styles from '../../styles/Form.module.css'
+import infoIcon from '../../public/images/form-info-icon.png'
+import britoFormImage from '../../public/images/brito-group-image-form.svg'
+import profilePictureTemplate from '../../public/images/brito-profile-picture-template.svg'
+import closeModalIcon from '../../public/images/close-modal-icon.svg'
+import logo from '../../public/images/logo-small.svg'
+
+const Form = () => {
+    const [fullName, setFullName] = useState('')
+    const [expertise, setExpertise] = useState('')
+    const [company, setCompany] = useState('')
+    const [resideInCanada, setResideInCanada] = useState('yes')
+    const [city, setCity] = useState('')
+    const [linkedinProfileLink, setLinkedinProfileLink] = useState('')
+    const [email, setEmail] = useState('')
+
+    Modal.setAppElement('#__next')
+
+    const [isProfilePictureModalOpen, setIsProfilePictureModalOpen] = useState(false)
+    const [isSuccesfullSubmissionModalOpen, setisSuccesfullSubmissionModalOpen] = useState(false)
+
+    function openProfilePictureModal() {
+        setIsProfilePictureModalOpen(true)
+    }
+
+    function closeProfilePictureModal() {
+        setIsProfilePictureModalOpen(false)
+    }
+
+    function openSuccessSubModal() {
+        setisSuccesfullSubmissionModalOpen(true)
+    }
+
+    function closeSuccessSubModal() {
+        setisSuccesfullSubmissionModalOpen(false)
+    }
+
+    const radioButtonHandler = (e) => {
+        setResideInCanada(e.target.value)
+    }
+
+    return (
+        <>
```

Comment:
The following div is already a wrapper, fragment no needed

Reply:
Done